### PR TITLE
Upgrade closure-templates to 2016-08-19

### PR DESCRIPTION
**Pull request by hochhaus**
The extra files from jssrc (other than soyutils_usegoog.js) are used in a change that I will send shortly.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: third_party/jssrc/BUILD [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # Apache
```
Comment (jart) [inline third_party/jssrc/BUILD]: Apache 2.0

**Comment by jart** [comment]
Thank you! Sorry I didn't get to this earlier. Feel free to ping me on chat any time.

Mr. Jenkins: test this please

File: third_party/jssrc/BUILD (hunk 1 above)
Comment (hochhaus) [inline third_party/jssrc/BUILD]: Fixed

**Comment by hochhaus** [comment]
Thanks. PTAL

**Comment by jart** [comment]
LGTM

Mr. Jenkins: test this please

**Pull request merged**